packages/client/src/components/DebitDetailsCard/DebitDetailsCard.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * @author David Zhu <dzhu292@aucklanduni.ac.nz>
+ */
+
+import Card from '../Card/Card';
+import DebitDetailsCardProps from './DebitDetailsCardProps';
+import { MdContentCopy } from 'react-icons/md';
+
+
+const DebitDetailsCard = (props: DebitDetailsCardProps) => {
+    const body = props.text.map((text, index) => 
+        <p className="font-normal text-gray-500" key={index}>{text}</p>); // TODO: redundant
+    if (typeof(props.sessionId) != "undefined") {
```

Comment:
This can actually be simplified down to `if (props.sessionId) ...` because if it is null or undefined it will be 'false'